templates/node/node--news--full.html.twig

```diff
@@ -0,0 +1,6 @@
+{% include "@molecules/page-title/page-title.twig" with {
+  page_title__heading: label,
+  page_title__meta: 'By ' ~ content.field_author|render ~ date,
```

Comment:
The author field is optional for news so we will want to make the 'By ' prefix conditional.

Reply:
Good catch. I pushed up a change that shows it if provided, and doesn't if it's not 👍